Assets/_BForBoss/_Entities/Scripts/HealingBehaviour.cs

```diff
@@ -0,0 +1,20 @@
+using UnityEngine;
+
+namespace Perigon.Entities
```

Comment:
To be honest, this doesnt seem like an Entity to me, nor general enough to put into our Perigon Namespace.

Reply:
I put this here because it's related to the lifeCycle, same with Damage being part of entity.

Reply:
I'm moving this to _core